docs/tutorials/platform_beginner/7_customize_login_and_register_forms.md

```diff
@@ -0,0 +1,212 @@
+# Step 7 - Customizing the user forms
+
+We want to have a consistent design over our web site. When using the pagelayout for the register forms, you see the design of your website. We want to welcome our new users with a pretty register page.
```

Comment:
`consistent design in our website`
Also maybe `you see the same design as the rest of the website`